app/assets/stylesheets/iframe.css.scss

```diff
@@ -23,7 +23,7 @@ body.iframe {
     width: 80%;
     margin: 0 auto 20px auto;
   }
-  #identity-register {
+  #identity-register, #ask-for-email {
```

Comment:
@nathanstitt this could use your blessing -- since I copied the styling from the normal CSS for this new "ask-for-email" form, I thought I should do copy the styling here too.